src/oc_erchef/apps/oc_chef_wm/src/oc_chef_wm_base.erl

```diff
@@ -953,7 +954,7 @@ verify_request_signature(Req,
              State#base_state{log_msg = {not_found, user_or_client}}};
         {error, Error} ->
             Msg = verify_request_message(error_finding_user_or_client, Name, OrgName),
-            {false, wrq:set_resp_body(chef_json:encode(Msg), Req),
+            {{halt, 500}, wrq:set_resp_body(chef_json:encode(Msg), Req),
```

Comment:
I'm wondering if we want a case specifically for no_connections where we return 503 so that we reserve 500 for "something unexpected went wrong".